snuba/consumer.py

```diff
@@ -58,10 +60,16 @@ def process_message(
             value = rapidjson.loads(message.payload.value)
         else:
             value = json.loads(message.payload.value)
-        metadata = KafkaMessageMetadata(
-            offset=message.offset, partition=message.partition.index
+
+        processed = self._process_message_impl(
+            value,
+            KafkaMessageMetadata(
+                offset=message.offset,
+                partition=message.partition.index,
+                timestamp=message.timestamp,
```

Comment:
Do you care of how we configured the broker message timestamp (whether it is the timestamp of message creation or when the message is appended to the log) ?
I think we use the default (created)

Reply:
I don't think it matters too much for the general case. For the measuring the internal latency specifically for subscriptions it does, obviously.